.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea should be in gitignore. Please exclude this file from commit

Reply:
Done